src/lightning_app/utilities/packaging/lightning_utils.py

```diff
@@ -83,6 +83,8 @@ def get_dist_path_if_editable_install(project_name) -> str:
     """Is distribution an editable install - modified version from pip that
     fetches egg-info instead of egg-link"""
```

Comment:
could we ask @tchaton for review on this one?

Reply:
I did, so pending now :)